Extension/browser/firefox/install.rdf

```diff
@@ -27,7 +27,7 @@
             <Description>
                 <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
                 <em:minVersion>30.0</em:minVersion>
-                <em:maxVersion>45.*</em:maxVersion>
+                <em:maxVersion>*</em:maxVersion>
```

Comment:
Oh, it's ok. I found a proof.
https://addons.mozilla.org/en-US/firefox/pages/appversions/